src/components/dialog/dialog.js

```diff
@@ -605,9 +605,7 @@ function MdDialogProvider($$interimElementProvider) {
 
         element.css('top', parentOffset + 'px');
 
-        options.backdrop = angular.element('<md-backdrop class="md-dialog-backdrop md-opaque">');
-        options.backdrop.css('top', parentOffset + 'px');
-        $mdTheming.inherit(options.backdrop, options.parent);
+        options.backdrop = $compile('<md-backdrop class="md-dialog-backdrop md-opaque">')(scope);
```

Comment:
Any reason for compiling this rather than just using `angular.element`?  It doesn't look to have anything dynamic in it.